routers/routes/routes.go

```diff
@@ -102,7 +117,9 @@ func NewMacaron() *macaron.Macaron {
 		loggerAsWriter := log.NewLoggerAsWriter("INFO", log.GetLogger("macaron"))
 		m = macaron.NewWithLogger(loggerAsWriter)
 		if !setting.DisableRouterLog && setting.RouterLogLevel != log.NONE {
-			log.SetupRouterLogger(m, setting.RouterLogLevel)
+			if log.GetLevel() <= setting.RouterLogLevel {
```

Comment:
So actually I've just noticed a bug. This should be looking at the router log-level not the general log-level...

```suggestion
			if log.GetLogger("router").GetLevel() <= setting.RouterLogLevel {
```